cmd/dns_list.go

```diff
@@ -57,37 +51,33 @@ Supported output template annotations: %s`,
 }
 
 func listDomains(filters []string) (outputter, error) {
-	domains, err := csDNS.GetDomains(gContext)
+	domains, err := cs.ListDNSDomains(gContext, gCurrentAccount.DefaultZone)
```

Comment:
```suggestion
	ctx := exoapi.WithEndpoint(gContext, exoapi.NewReqEndpoint(gCurrentAccount.Environment, gCurrentAccount.DefaultZone))
	domains, err := cs.ListDNSDomains(ctx, gCurrentAccount.DefaultZone)

```

Without this, requests will fail in other envs.

Reply:
Ah yes, good catch. I'll fix it.